### DISCUSSION TITLE
Commonsense Knowledge, Symbolic Knowledge Distillation paper: Existing resources

### DISCUSSION THREAD

**Discussion by mayanks43** [eduardoblanco/csc696H-Spring24#13] [Q&A]
"Yet, models are trained on existing resources; ATOMIC10x is generated without these." Is this a valid statement to make? Is ATOMIC10x not generated on existing resources?

**Comment by Abby-OGV** [comment]
Not quite. 
$ATOMIC^{10X}$ uses data from $`ATOMIC^{20}_{20}`$ in its prompts.  For example, "We use 100 high-quality events from
the $ATOMIC^{20}_{20}$ for our prompt,..."

So, it's a stretch to say that $ATOMIC^{10X}$ is generated without existing resources.

**Comment by eduardoblanco** [comment]
I agree. They would probably argue that they use a very tiny portion or ATOMIC.

It is true that they are generating new tail events (new == not in ATOMIC), correct?
